internal/testutil/helpers/helpers.go

```diff
@@ -4,63 +4,21 @@
 // not use this file except in compliance with the License. You may obtain
 // a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
 
-package testhelpers // import "go.mongodb.org/mongo-driver/internal/testutil/helpers"
+package helpers
 
 import (
-	"fmt"
 	"io/ioutil"
-	"math"
 	"path"
-	"strings"
-	"time"
-
 	"testing"
 
-	"io"
-
-	"reflect"
-
 	"github.com/stretchr/testify/require"
 	"go.mongodb.org/mongo-driver/bson"
-	"go.mongodb.org/mongo-driver/x/mongo/driver/connstring"
 )
 
-// Test helpers
-
-// IsNil returns true if the object is nil
-func IsNil(object interface{}) bool {
-	if object == nil {
-		return true
-	}
-
-	value := reflect.ValueOf(object)
-	kind := value.Kind()
-
-	// checking to see if type is Chan, Func, Interface, Map, Ptr, or Slice
-	if kind >= reflect.Chan && kind <= reflect.Slice && value.IsNil() {
-		return true
-	}
-
-	return false
-}
-
-// RequireNotNil throws an error if var is nil
-func RequireNotNil(t *testing.T, variable interface{}, msgFormat string, msgVars ...interface{}) {
-	if IsNil(variable) {
-		t.Fatalf(msgFormat, msgVars...)
-	}
-}
-
-// RequireNil throws an error if var is not nil
-func RequireNil(t *testing.T, variable interface{}, msgFormat string, msgVars ...interface{}) {
```

Comment:
Some strange helpers here; makes much more sense to just use the `require` package...

Reply:
Agreed. I'm assuming again that these are vestigial and were never removed when we started using the [stretchr/testify](https://github.com/stretchr/testify) package and the internal `assert` package.